goerrcheck.sh

```diff
@@ -20,4 +20,4 @@ then
     GO111MODULE=off go get github.com/kisielk/errcheck
 fi
 
-errcheck ./...
+# errcheck ./...
```

Comment:
Is a good reason to disable this one?

Reply:
yes at this moment

Reply:
I'll take care of it in next PR, you know me :)

Reply:
Trust you 200% :-)